src/redux/actions/cellSets/loadCellSets.js

```diff
@@ -5,15 +5,19 @@ import {
 
 const loadCellSets = (experimentId) => async (dispatch, getState) => {
   const {
-    loading, error,
+    loading, error, updatingClustering,
   } = getState().cellSets;
 
-  if (!loading && !error) {
+  if ((!loading && !error) || updatingClustering) {
```

Comment:
why are we calling this action at all when doing cell set reclustering?

Reply:
It is a case I ran into when testing the application with no louvain clustering.

Both configure embedding and some of the plots call loadCellSets, so instead of handling it everywhere I felt it was better to add a guard to the already existing ones.

Reply:
yeah, adding the guard here sounds more sensible than having to modify each of the individual ones